src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapMercury.java

```diff
@@ -54,8 +55,49 @@ public void vuapCheckTopAreasMercury(Page page, String slot) {
 
   @Test(
       dataProviderClass = MobileAdsDataProvider.class,
-      dataProvider = "adsVuapResolvedStateMobile",
-      groups = {"AdsVuapTimeProgressMercury"}
+      dataProvider = "adsVuapMobile",
+      groups = "AdsVuapDefaultStateMercury"
+  )
+  public void vuapDefaultStateShouldStartPlayingAdvertisementAutomatically(Page page, String slot) {
+    AdsBaseObject ads = new AdsBaseObject(driver, page.getUrl());
+    final AutoplayVuap vuap = new AutoplayVuap(driver, slot, ads.findFirstIframeWithAd(slot), false);
+
+    ads.scrollToSlot(slot);
+    vuap.waitForVideoStart();
+
+    VuapAssertions.verifyVideoPlay(vuap);
+  }
+
+  @Test(
+      dataProviderClass = MobileAdsDataProvider.class,
+      dataProvider = "adsVuapMobile",
+      groups = {"AdsVuapDefaultStateMercury"}
+  )
+  public void vuapDefaultStateReplayIsNotMuted(Page page, String slot) throws InterruptedException {
+    AdsBaseObject ads = openPageWithVideoInLocalStorage(page, VuapVideos.VIDEO_10s);
+    final AutoplayVuap vuap = new AutoplayVuap(driver, slot, ads.findFirstIframeWithAd(slot), false);
+    ads.scrollToSlot(slot);
+
+    // 2017-10-12 scrollTo takes so much time that the video for sure has started and ended
+    /*
+    vuap.waitForVideoStart();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)